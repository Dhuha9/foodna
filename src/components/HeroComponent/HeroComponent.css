@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');

* {
    box-sizing: border-box;
}

.bg{
  background-color: #03B664;
  color: white;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  min-height: 100vh;
  margin: 0;
  font-family: 'Roboto', sans-serif;
  overflow: hidden;
}

.side-crop { 
    max-height: 400px; 
    overflow: none;
    margin-bottom: 0px;
}

.side-crop img{
    width: 800px;
    height : auto;
    left: 320px;
    top: 353px;
    filter: drop-shadow(4px 4px 9px rgba(0, 0, 0, 0.25));
    border-radius: 5px;
    margin-top:40px;
    bottom: 0px;
}

h1{
    width: 787px;
    height: 75px;
    left: calc(50% - 787px/2 - 0.5px);
    top: 51px;
    font-weight: 500;
    font-size: 64px;
    line-height: 75px;
    text-align: center;
}

p{
    width: 682px;
    height: 62px;
    font-size: 20px;
    line-height: 140.62%;
    text-align: center;

}