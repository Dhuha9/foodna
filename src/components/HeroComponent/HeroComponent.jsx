import React from 'react'
import CtaButton from '../CtaButton/CtaButton';
import './HeroComponent.css';

export default function HeroComponent() {
    return (
        <div className="bg">
            <h1>You have Extra food ?</h1>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
            <CtaButton />
            <div className="side-crop">
                <img className="hero-pic" src="img/apple.jpg" alt="" />
            </div>
            
        </div>
    )
}
