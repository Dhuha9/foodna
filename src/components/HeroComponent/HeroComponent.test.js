import React from 'react';
import renderer from 'react-test-renderer';
import HeroComponent from './HeroComponent';
import { BrowserRouter as Router } from 'react-router-dom';

it('renders when there are is no HeroComponent', () => {
  const tree = renderer
    .create(
      <Router>
        <HeroComponent />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
