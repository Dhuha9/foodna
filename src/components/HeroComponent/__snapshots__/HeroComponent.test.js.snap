// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there are is no HeroComponent 1`] = `
<div
  className="HeroComponentStyle"
>
  <div
    className="bg"
  >
    <h1>
      HeroComponent.headline
    </h1>
    <p>
      HeroComponent.paragraph
    </p>
    <a
      href="/donate"
      onClick={[Function]}
    >
      <button
        className="btn btn-default"
        style={
          Object {
            "backgroundColor": "#FFB701",
            "border": "none",
            "borderRadius": "5px",
            "color": "#000",
            "fontSamily": "roboto",
            "fontSize": "18px",
            "height": "60px",
            "left": "calc(50% - 240px/2 - 1px)",
            "top": "256px",
            "width": "240px",
          }
        }
      >
        CtaButton.title
      </button>
    </a>
    <div
      className="side-crop"
    >
      <img
        alt=""
        className="hero-pic"
        src="img/apple.jpg"
      />
    </div>
  </div>
</div>
`;
